terraform/account/opensearch.tf

```diff
@@ -0,0 +1,197 @@
+data "aws_vpc_endpoint" "opensearch" {
+  tags = {
+    Name = "opensearch-eu-west-1"
+  }
+  provider = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_security_policy" "lpas_collection_encryption_policy" {
+  name        = "policy-${local.account_name}"
+  type        = "encryption"
+  description = "encryption policy for collection"
+  policy = jsonencode({
+    Rules = [
+      {
+        Resource     = ["collection/collection-${local.account_name}"],
+        ResourceType = "collection"
+      }
+    ],
+    AWSOwnedKey = false
+    KmsARN      = aws_kms_alias.opensearch_alias_eu_west_1.target_key_arn
+  })
+  provider = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_collection" "lpas_collection" {
+  name       = "collection-${local.account_name}"
+  type       = "SEARCH"
+  depends_on = [aws_opensearchserverless_security_policy.lpas_collection_encryption_policy]
+  provider   = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_security_policy" "lpas_collection_network_policy" {
+  name        = "policy-${local.account_name}"
+  type        = "network"
+  description = "VPC access for collection endpoint"
+  policy = jsonencode([
+    {
+      Description = "VPC access for collection endpoint",
+      Rules = [
+        {
+          ResourceType = "collection",
+          Resource     = ["collection/collection-${local.account_name}"]
+        }
+      ],
+      AllowFromPublic = false,
+      SourceVPCEs = [
+        data.aws_vpc_endpoint.opensearch.id
+      ]
+    },
+    {
+      AllowFromPublic = true
+      Description     = "public access to dashboard"
+      Rules = [
+        {
+          Resource     = ["collection/collection-${local.account_name}"]
+          ResourceType = "dashboard"
+        }
+      ]
+    }
+  ])
+  provider = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_access_policy" "team_operator_access" {
+  count       = local.account_name == "production" ? 0 : 1
+  name        = "team-access-${local.account_name}"
+  type        = "data"
+  description = "allow index and collection access for team"
+  policy = jsonencode([
+    {
+      Rules = [
+        {
+          ResourceType = "index",
+          Resource     = ["index/collection-${local.account_name}/*"],
+          Permission   = ["aoss:*"]
+        },
+        {
+          ResourceType = "collection",
+          Resource     = ["collection/collection-${local.account_name}"],
+          Permission   = ["aoss:*"]
+        }
+      ],
+      Principal = [
+        "arn:aws:iam::${data.aws_caller_identity.global.account_id}:role/operator"
+      ]
+    }
+  ])
+  provider = aws.eu_west_1
+}
+
+resource "aws_opensearchserverless_access_policy" "team_breakglass_access" {
```

Comment:
Would we always want this available on production? Or have we decided opensearch is always off-limits for live data?

Reply:
Oh ignore me, I've just seen the ordering is different here for `count` 🙄

Reply:
Breakglass is an alarmed role (using it raises an alarm) for use in emergencies.